src/index.js

```diff
@@ -1559,6 +1627,99 @@ Compute.prototype.getImages = function(options, callback) {
   );
 };
 
+/**
+ * Get a list of instance templates.
+ *
+ * @see [InstanceTemplates Overview]{@link https://cloud.google.com/compute/docs/instanceTemplates}
```

Comment:
https://cloud.google.com/compute/docs/instance-templates/ I believe. Why do they keep changing the links? :(